src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
```

Comment:
Are you sure you need this? You are trimming again within `reduce` so probably this one is redundand